### ISSUE TITLE
NVDA's autoupdate conflicts with Avira's scanner

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2789]
**Reported by andre_baldo on 2012-11-08 20:31**
If Avira Antivirus is on, NVDA cannot be autoupdated. Avira says that the file was not directed to its scanner.

**Comment by ehollig** [comment]
If I recall from previous closed issues, there is nothing NVDA can do about antiviruses flagging NVDA as a false positive. Is this still true in this case?
Also, it appears this issue is over 5 years old since it was reported.

**Comment by Brian1Gaff** [comment]
Hi about a year ago, I managed to get a scan running and had no issues at 
all in that software, other than I hated it and it took ages, so went back 
to Microsoft again.
 It seems to flag lots of things up which are just ordinary tools as well, 
so its entirely possible it might act  up again.
 These issues need to really be reported to the anti virus maker as its 
seldom  an NVDA problem, indeed I've never had a problem with bona fide' 
nvda code and add ons.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----
